packages/config/src/api/site_info.ts

```diff
@@ -37,30 +36,58 @@ export const getSiteInfo = async function ({
 }: GetSiteInfoOpts) {
   const { env: testEnv = false } = testOpts
 
-  if (api === undefined || mode === 'buildbot' || testEnv) {
+  if (api === undefined || testEnv || offline) {
```

Comment:
I'd be inclined not to change this logic at all, and use the feature flag to go into a full new set of logic.

Reply:
It has to change sadly as I don't want buildbot requests hitting this `if` where we want to be using the v2 route. Reason we can't read the flag first before all this logic is that we have to call `getSite` which tells us what feature flags there are for that site.